atomic_reactor/plugins/pre_bump_release.py

```diff
@@ -87,7 +113,7 @@ def run(self):
             if not build:
                 break
```

Comment:
That seems correct. Can we make sure the unit tests are reflecting this properly now? IIRC, they were passing with the previous behavior as well.